static/app/views/starfish/components/tableCells/renderHeadCell.tsx

```diff
@@ -72,3 +85,24 @@ export const getAlignment = (key: string): Alignments => {
   }
   return 'left';
 };
+
+function getFieldDescription(key: string, isAggregate: boolean): string | undefined {
+  if (isAggregate) {
+    switch (key) {
+      case `${SPM}()`:
+        return t('The number of queries per minute across all endpoints.');
+      case `avg(${SPAN_SELF_TIME})`:
+        return t('The average duration of this query across all endpoints.');
+      default:
```

Comment:
I think you can use `getSpanOperationDescription` to generalize this! Otherwise, it'll says `"this query"` even for other span types